### PR TITLE
Release 26.2 changelog

### PR DESCRIPTION
**Pull request by lucasmoura**
Add changelog for release 26.2

### CODE REVIEW

**Review by mitechie — APPROVED** [review]
One comment I understand is something previously landed but still wanted to check.

File: uaclient/tests/test_config.py [hunk 1]
```diff
@@ -529,8 +529,11 @@ class TestStatus:
     esm_desc = ENTITLEMENT_CLASS_BY_NAME["esm-infra"].description
     cc_eal_desc = ENTITLEMENT_CLASS_BY_NAME["cc-eal"].description
 
-    def check_beta(self, cls, show_beta, uacfg=None):
+    def check_beta(self, cls, show_beta, uacfg=None, status=""):
         if not show_beta:
+            if status == "enabled":
```
[the remark this answers is not in the document]
Comment (lucasmoura) [reply uaclient/tests/test_config.py:534]: yes, it should be. I update the code to use the correct enum value, but missed this testing bit. I will address this on a new PR.

Thanks for spotting this @mitechie

**Review by blackboxsw — CHANGES_REQUESTED** [review]
As mentioned in side-channel The commit message on 76b863512e3642af695a8046cc16cea14f43aed4 had the summary
     "backport: show beta services in status if enabled"
This was actually a cherry-pick I think. so we don't want to label it as backport because our release of 27.0 will try to revert any commit with the "backport:" prefix.

**Review by blackboxsw — APPROVED** [review]
LGTM!

**Pull request merged**